.ci/linux/cpeditor.desktop

```diff
@@ -1,7 +1,16 @@
 [Desktop Entry]
+Version=1.0
 Type=Application
 Name=CP Editor
-Comment=The Editor for Competitive Programming
-Exec=cpeditor
-Icon=cpeditor-icon
-Categories=Development;
+GenericName=Competitive Programming Editor
+GenericName[ru]=Редактор кода для спортивного программирования
+GenericName[zh_CN]=竞争编程编辑器
```

Comment:
Are you sure we should have a long name? "CP" comes from "Competitive Programming", but I think using "Competitive Programming Editor“ as a long name of CP Editor is weird.